helm/opendistro-es/user-roles-values.yaml

```diff
@@ -0,0 +1,227 @@
+certmanager:
+  enabled: true
+  elasticsearch:
+    transport:
+      commonName: nodes.elastic-system.svc.cluster.local
+      organization:
+      - Elastisys
+    admin:
+      enabled: true
+      commonName: admin.elastic-system.svc.cluster.local
+      organization:
+      - Elastisys
```

Comment:
Is this necessary for user configuration? If it can be omitted, please do so. That makes it easier to see what is relevant.

Reply:
The `commonName` and `organization`  are used for admin configuration  under ` config: `block in elasticsearch.

Reply:
Yeah, but it is not relevant for configuring roles and users.

Reply:
I will remove it! I thought it would make the config complete :)